app/namespaces/data_provider/data_provider_controller.py

```diff
@@ -58,7 +58,8 @@ def post(self):
                                       prioritize_estimates_mode=prioritize_estimates_mode,
                                       include_in_srma=include_in_srma,
                                       include_disputed_regions=include_disputed_regions,
-                                      include_subgeography_estimates=include_subgeography_estimates)
+                                      include_subgeography_estimates=include_subgeography_estimates,
+                                      unity_aligned=unity_aligned)
```

Comment:
maybe we could rename this parameter to be slightly more clear? include_only_unity_aligned or unity_aligned_only

Reply:
let's do `unity_aligned_only` don't usually love long parameter names but this is justified